runtime_lib/CMakeLists.txt

```diff
@@ -13,22 +13,71 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 project("aie runtime libs")
 
-# include(ExternalProject)
+include(ExternalProject)
 
-# set(INSTALLS
-#     test_library.cpp
-#     test_library.h)
+# test_library
+# FIXME: don't ship as source... compile them instead.
```

Comment:
@denolf The FIXME here was somewhat important... I was trying to limit this to having one external project for all of the runtime_lib directory.  Do you have a better idea for how to accomplish that without having the external project in the toplevel CMakeLists.txt?